pkg/psalabelsyncer/podsecurity_label_sync_controller_test.go

```diff
@@ -787,6 +802,7 @@ func TestPodSecurityAdmissionLabelSynchronizationController_sync(t *testing.T) {
 			if nsModified != nil && len(tt.expectedPSaLevel) > 0 {
 				require.Equal(t, tt.expectedPSaLevel, nsModified.Labels[psapi.WarnLevelLabel], "unexpected PSa warn level")
 				require.Equal(t, tt.expectedPSaLevel, nsModified.Labels[psapi.AuditLevelLabel], "unexpected PSa audit level")
+				require.Equal(t, tt.expectedAnnotation, nsModified.Annotations[securityv1.MinimallySufficientPodSecurityStandard], "unexpected PSa annotation value")
```

Comment:
Should this be moved to a conditional for the `expectedAnnotation` field rather than the `expectedPSaLevel`?